pages/random.js

```diff
@@ -0,0 +1,25 @@
+import { withoutAuth } from "/components/Auth";
+import { useRouter } from "next/router";
+
+function Random() {
+  const router = useRouter();
+  const memeLinks = [
+    "https://www.youtube.com/watch?v=dQw4w9WgXcQ",
+    "https://www.youtube.com/watch?v=WaaANll8h18",
+    "https://www.youtube.com/watch?v=tas0O586t80",
+    "https://www.youtube.com/watch?v=bUo1PgKksgw",
+    "https://www.youtube.com/watch?v=nbWqLK-YPT4",
+    "https://www.youtube.com/watch?v=OMa9iVObGQc",
```

Comment:
```suggestion
    "https://www.youtube.com/watch?v=OMa9iVObGQc",
    "https://www.youtube.com/watch?v=Giotzu7TJj0",
    "https://www.youtube.com/watch?v=bCkCWnmAD-o",
    "https://www.youtube.com/watch?v=Evz_Hxi96e8"
```

I think you are missing a couple of them :point_up: